persistence-modules/hibernate-5-2/src/test/java/com/baeldung/hibernate_parameters/Event.java

```diff
@@ -0,0 +1,30 @@
+package com.baeldung.hibernate_parameters;
```

Comment:
Two things here: let's put this under src/main/java, and remove the underscore from the package name.